docs/3.tutorials/nfts/9-series.md

```diff
@@ -0,0 +1,428 @@
+
+We've also added a field `tokens` which keeps track of all the token IDs that have been minted for this series. This allows us to deal with the potential `copies` cap by checking the length of the set. It also allows us to paginate through all the tokens in the series.
+
+### Series File
+
+Moving on to a new file that has been introduced, you'll notice that there's no longer the `mint.rs` file and it's been replaced by a newer `series.rs` file. This is responsible for everything related to creating series as well as minting NFTs. Starting with the logic for creating a new series, you'll see a new function `create_series`.
+
+```rust reference
+https://github.com/near-examples/nft-tutorial/blob/main/nft-series/src/series.rs#L7-L58
+```
+
+The function takes in a series ID in the form of a `u64`, the metadata, royalties, and the price for tokens in the series. It will then create the `Series` object and insert it into the contract's `series_by_id` data structure. It's important to note that the caller must be an approved creator and they must attach enough $NEAR to cover storage costs.
+
+#### Minting
+
+Next, we'll look at is the minting function. If you remember from before, this used to take a slew of parameters:
+- Token ID
+- Metadata
+- Receiver ID
+- Perpetual Royalties 
+
+With the new minting function, these parameters have been brought down to just two.
+- The series ID
+- The receiver ID.
+
+Notice how the token ID isn't required? This is because the token ID is automatically generated when minting. The ID is stored on the contract as `${series_id}:${token_id}` where the token ID is a nonce that increases each time a new token is minted in a series. This not only reduces the amount of information stored on the contract but it also acts as a way to check the specific edition number.
+
+The mint function will charge users for storage and if the series has a price, it will ensure that users attach enough $NEAR to cover the price **and** storage. At the end, it will then send all the $NEAR attached minus the storage costs to the **series owner**. It's important to note that in this case, the caller does **not** need to be an approved minter.
+
+If a price wasn't specified, the $NEAR attached will be used to pay for storage and any excess will be refunded to the caller. In addition, all the callers must be approved minters.
+
+```rust reference
+https://github.com/near-examples/nft-tutorial/blob/main/nft-series/src/series.rs#L60-L149
+```
+
+### View Functions
+
+Now that we've introduced the idea of series, more view functions have been added to support the potentially complex use cases that arise with this contract. With these functions, a new `JsonSeries` struct has been added due to the fact that you cannot serialize an `UnorderedSet` which is being stored in each series. 
```

Comment:
```suggestion
Now that you are using a `Series` structure for your NFT contract, retrieving on-chain data from these contracts will require you to create new `view` methods. 

Here we will use a new struct (`JsonSeries`) that will store the metadata for each series.
```

Reply:
Lmk if line 165 is accurate. Explaining the serialization of an unordered set might be unnecessary here and more appropriate in the area where we outline the `JsonSeries` struct.